### ISSUE TITLE
Disable not working in nested includes DeclarationOrder

### ISSUE THREAD

**Issue by kaelig** [causes/scss-lint#355]
An include in an include throws an error even when the linters are disabled:

```scss
// file.scss

// scss-lint:disable

@include mixin(blah) {
	.el { @include mixin; }

	@include mixin;
}
```

```
file.scss:4 [W] DeclarationOrder: Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets
```

**Comment by sds** [comment]
Thanks for the bug report, @kaelig!

Fixed in ccafdd92d6d7de8ba569711c5ed166824b88383c.
